src/components/Avatar/Avatar.style.ts

```diff
@@ -0,0 +1,84 @@
+import { styled } from 'styled-components';
+
+export type Shape = 'circle' | 'round' | 'square';
+export type Size = 'XS' | 'S' | 'M' | 'L' | 'XL';
+
+export interface AvatarProps {
+  size?: Size;
+  shape?: Shape;
+  src?: string;
+  isBorder?: boolean;
+  isShadow?: boolean;
+  isEdit?: boolean;
+  onClick?: (event: React.MouseEventHandler<HTMLElement>) => void;
+}
+
+const ShapeToStyle: { [key: string]: string } = {
+  circle: '50%',
+  round: '5px',
+  square: '0px',
+};
+
+const SizeToStyle: { [key: string]: string } = {
+  XS: '26px',
+  S: '36px',
+  M: '46px',
+  L: '56px',
+  XL: '66px',
+};
+
+export const AvatarWrapper = styled.div<{
+  size?: Size;
+  shape?: Shape;
+  src?: string;
+  $isBorder?: boolean;
+  $isShadow?: boolean;
+}>`
+  position: relative;
+  box-sizing: border-box;
+  display: inline-block;
+  width: ${({ size }) => size && SizeToStyle[size]};
+  height: ${({ size }) => size && SizeToStyle[size]};
+  overflow: hidden;
+  border: ${({ $isBorder, theme }) =>
+    $isBorder ? `1px solid ${theme.transparent_30}` : 'none'};
+  border-radius: ${({ shape }) => shape && ShapeToStyle[shape]};
```

Comment:
```suggestion
  width: ${({ size }) => SizeToStyle[size]};
  height: ${({ size }) => SizeToStyle[size]};
  overflow: hidden;
  border: ${({ $isBorder, theme }) =>
    $isBorder ? `1px solid ${theme.transparent_30}` : 'none'};
  border-radius: ${({ shape }) => ShapeToStyle[shape]};
```

props의 && 연산자 부분을 모두 제거할 수 있게됩니다!

Reply:
예제 코드 감사합니다!